### ISSUE TITLE
Can not activate the extension in ubuntu 20.04

### ISSUE THREAD

**Issue by huwei9527** [taosdata/vscode-tdengine#3]
Vscode:

Version: 1.74.3
Commit: 97dec172d3256f8ca4bfb2143f3f76b503ca0534
Date: 2023-01-09T16:57:40.428Z
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Linux x64 5.15.0-57-generic
Sandboxed: No

vscode-engine version: v0.9.1

**Comment by wjsvec** [comment]
> 

Received this problem, I will try to repeat this error later.

The development is based on the 1.71 version of vscode and is on the win system. I need some time to try my performance on ubuntu

**Comment by wjsvec** [comment]
@sangshuduo @huolibo
I would like to be assigned to this issue. Thanks a lot.

**Comment by wjsvec** [comment]
@huwei9527 Please provide a screenshot or more information of the error. It should be ok in ubuntu

**Comment by wjsvec** [comment]
Yes, the extension can not activate in ubuntu, I will deal with this problem later.
